getting-started-tutorial/01-mlrun-basics.ipynb

```diff
@@ -399,24 +399,52 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 10,
+   "execution_count": 28,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "## Set this environment variable in case of dark site\n",
+    "# os.environ['SAMPLE_DATA_SOURCE_URL_PREFIX'] = '/v3io/bigdata/demos-data/iguazio/'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 30,
    "metadata": {},
    "outputs": [],
    "source": [
     "# Set the source-data URL\n",
-    "source_url = 'https://s3.wasabisys.com/iguazio/data/iris/iris.data.raw.csv'"
+    "url_prefix = os.environ.get('SAMPLE_DATA_SOURCE_URL_PREFIX', 'https://s3.wasabisys.com/iguazio/')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 31,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "source_url = f'{url_prefix}data/iris/iris.data.raw.csv'"
```

Comment:
We need to consider what happens if we don't have / at the end of url_prefix. So we need the code the work the same if we have a trailing '/' or if we don't have.
So I think this is better source_url = f"{url_prefix.strip('/')}/data/iris/iris.data.raw.csv"

Reply:
Modifed, used `rstrip` instead